.github/workflows/build-test-deploy.yml

```diff
@@ -29,8 +28,19 @@ permissions:
   pull-requests: write
 
 jobs:
+  set-runner:
```

Comment:
What issue does this solve? Seems to me this makes our workflow unnecessarily more complex?

I also don't see this back in the GitHub documentation. Why not leave it as it currently is?

Reply:
zie bv https://stackoverflow.com/questions/71961921/specify-runner-to-be-used-depending-on-condition-in-a-github-actions-workflow

De env var op de runner werkt niet blijkt (vreemd dat ik dat niet eerder heb gezien) en
anders kan ik het beter weer weg halen. Dan zetten we de runner op elke job weer terug naar ubuntu-latest. Dit is ook niet zo'n groot ding, het is enkel om 1 explicite versie van de runner te kunnen zetten...

Reply:
I mean: I would just go back to `runs-on: ubuntu-latest` like we had really. Not such a big deal. Or `ubuntu-22.04` if we wish to be explicit. Dependabot will make PRs to update these anyway. :-)

Reply:
Yes, let's do that - this does make it more complex than needed